### PR TITLE
[acl]: Update documentation

### PR DESCRIPTION
**Pull request by gavindidrichsen**
Signed-off-by: Gavin Didrichsen <gavin.didrichsen@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,59 @@
-[![Build Status](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_apis/build/status/chef-base-plans.grep?branchName=master)](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_build/latest?definitionId=69&branchName=master)
+[![Build Status](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_apis/build/status/chef-base-plans.acl?branchName=master)](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_build/latest?definitionId=69&branchName=master)
```
Comment (MindNumbing) [inline README.md]: This still links to grep the following needs to change:

> definitionId=69

File: README.md [hunk 2]
```diff
@@ -1,21 +1,59 @@
...
+
+To install this plan, you should run the following commands to first install, and then link the binaries this plan creates.
+
+``hab pkg install core/acl --binlink``
+
+will add the following binaries to the PATH:
+
+* /bin/getfacl
+* /bin/chacl
+* /bin/setfacl
```
Comment (MindNumbing) [inline README.md:41]: Can you insert the stdout when installing / binlinking.

**Review by MindNumbing — CHANGES_REQUESTED** [review]
Couple of things to look at. Other than those it looks good.

**Review by MindNumbing — CHANGES_REQUESTED** [review]
Also, can you add 'inspec.lock' to the .gitignore

**Review by MindNumbing — APPROVED** [review]
Thanks @gavindidrichsen, LGTM! 👍

File: README.md (hunk 1 above)
Comment (gavindidrichsen) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (gavindidrichsen) [reply README.md:41]: Done.

**Pull request merged**